provider/app.go

```diff
@@ -77,6 +111,15 @@ func appResource() *schema.Resource {
 				ForceNew: true,
 				Optional: true,
 			},
+			"relative_path": {
```

Comment:
We should make sure this gets removed in all of our templates then!

Reply:
I just moved this value to be below subdomain, it's already been deprecated for a while now. I deprecated it a few weeks ago when we added the subdomain key.